learning_resources_search/serializers.py

```diff
@@ -76,11 +48,16 @@ def serialize_learning_resource_for_update(
         dict: The serialized and transformed resource data
 
     """
+    serialized_data = LearningResourceSerializer(instance=learning_resource_obj).data
+    if learning_resource_obj.resource_type == LearningResourceType.course.name:
+        serialized_data["course"]["course_numbers"] = [
+            SearchCourseNumberSerializer(instance=num).data
```

Comment:
Add back the `course_number` fields that are omitted by `CourseNumberSerializer`